app/decorators/identity_decorator.rb

```diff
@@ -1,9 +1,32 @@
 IdentityDecorator = Struct.new(:identity) do
+  include ActionView::Helpers::TagHelper
+
   def pretty_event_type
-    I18n.t('event_types.authenticated_at', service_provider: identity.display_name)
+    service_provider_html = if return_to_sp_url.present?
+                              return_to_sp_link
+                            else
+                              identity.display_name
+                            end
+    I18n.t('event_types.authenticated_at', service_provider: service_provider_html).html_safe
   end
 
   def happened_at
     EasternTimePresenter.new(identity.last_authenticated_at).to_s
   end
+
+  private
+
+  def return_to_sp_link
+    content_tag(
+      :a,
+      identity.display_name,
+      href: return_to_sp_url,
+      target: '_blank',
```

Comment:
This may just be me, as I tend to get a bit worked up over _blank, but I think we should think carefully about our use of it. There have been security problems with it, and it presents a lot of issues with usability, especially on mobile devices. From the research I've done, it appears that current wisdom is to reserve it for cases in which leaving the page would interrupt the user in an unrecoverable way -- like when they're in a complex workflow, or in the middle of a large video, and you cannot return them to their prior location if they hit "back".

On the other hand, if everyone feels this is just me getting worked up over nothing, that's OK too.

Reply:
I support this, if I understand correctly, the purpose of this PR is to get people to the SPs, and not to stay on login.gov so I think that it seems fine to make these regular links and not `target=_blank` ones